BParkingNano/plugins/BToKLLBuilder.cc

```diff
@@ -244,6 +250,12 @@ void BToKLLBuilder::produce(edm::StreamID, edm::Event &evt, edm::EventSetup cons
     } // for(size_t ll_idx = 0; ll_idx < dileptons->size(); ++ll_idx) {
   } // for(size_t k_idx = 0; k_idx < kaons->size(); ++k_idx)
 
+  for (auto & cand: *ret_val){
+    cand.addUserInt("n_k_used", std::count(used_trk_id.begin(),used_trk_id.end(),cand.userInt("k_idx")));
+    cand.addUserInt("n_l1_used", std::count(used_lep1_id.begin(),used_lep1_id.end(),cand.userInt("l1_idx"))+std::count(used_lep2_id.begin(),used_lep2_id.end(),cand.userInt("l1_idx")));
+    cand.addUserInt("n_l2_used", std::count(used_lep1_id.begin(),used_lep1_id.end(),cand.userInt("l2_idx"))+std::count(used_lep2_id.begin(),used_lep2_id.end(),cand.userInt("l2_idx")));
+  }
+
```

Comment:
what is the goal of this? 
for each candidate, this saves how many times each l1, l2, K is used over all the candidates...
Do you plan to use this information later on to clean the nano-producer? or is it to help the analysis?

Reply:
It is essentially for selection. The plot I sent in the pdf was from this variables